### ISSUE TITLE
Explain how to contribute

### ISSUE THREAD

**Issue by cimm** [brug-be/rubybelgium#24]
The [http://rubybelgium.be](RubyBelgium) website currently does not explain how to contribute. I think it would be nice to show we are an open group and value contribution. A contribute banner, link in the footer, button, I don't care.

This would especially help the newly added http://rubybelgium.be/rubyshops page as it's not clear where that data is coming from (why is there a Bruxelles and Brussels section for example) or how to add your own shop when this is how you find out there is a Belgian Ruby group.

**Comment by otagi** [comment]
Good idea. I was first thinking about a README. But I think a web page would be more appealing.

Points to list:

- The website itself ([README](https://github.com/brug-be/rubybelgium/blob/master/README.md#contributing))
- Ruby shops
- BRUG ([howto](https://github.com/brug-be/rubybelgium-documents/blob/master/events/howto_organize_a_brug.md))
- RubyCamp ([howto](https://github.com/brug-be/rubybelgium-documents/blob/master/events/howto_organize_a_rubycamp.md))
- RubyBurgers ([howto](https://github.com/brug-be/rubybelgium-documents/blob/master/events/howto_organize_a_rubyburger.md))
- RailsGirls

Anything else?

**Comment by toch** [comment]
The association document and board (i think the repo should be renamed though)

**Comment by borisrorsvort** [comment]
There is a lot to learn from those guys I think also http://www.opentechschool.org/handbooks/

**Comment by toch** [comment]
So we need someone to lead that :). @cimm don't you want to take over it?

**Comment by dantesolis** [comment]
Hey guys can I assign this to myself?
- [ ] Add Contribute Page
- [ ] Add Contribute banner/footer
- [ ] Explain how to contribute

**Comment by borisrorsvort** [comment]
@dantesolis Not yet. I'm rethinking the whole content of the site and communication strategy. The press page will have to wait as well and its far from finished imho. Also there is a new layout coming https://github.com/brug-be/rubybelgium/issues/50. For the press page we should still discuss what going in there as I suggested in the issue.

**Comment by toch** [comment]
@borisrorsvort Couldn't we do an intermediary version with the current version of the website? That would allow @dantesolis to start to contribute and step forward by doing. That would also help you to figure out what we need step by step. What do you think?

**Comment by borisrorsvort** [comment]
@toch switching discussion here https://github.com/brug-be/rubybelgium/issues/27

**Comment by borisrorsvort** [comment]
Ok so at this point, I think it makes the most sense to add a contribute section to each of the page instead of a separate page for it. It will be less scary imho
